apps/judicial-system/web/src/routes/Prosecutor/Indictments/Defendant.tsx

```diff
@@ -250,7 +251,11 @@ const Defendant: React.FC = () => {
         <Box component="section" marginBottom={5}>
           <Box marginBottom={3}>
             <Text as="h3" variant="h3">
-              {capitalize(formatMessage(core.indictmentDefendant))}
+              {capitalize(
+                formatMessage(core.indictmentDefendant, {
+                  gender: Gender.MALE,
```

Comment:
Should the gender be hardcoded?

Reply:
I am not sure if we should be changing the title of the page and the sidebar when the genre changes. What do you think?

Reply:
Not sure either. I will approve and we'll see if we get feedback.